src/vma/sock/sockinfo_tcp.cpp

```diff
@@ -3326,13 +3328,15 @@ int sockinfo_tcp::setsockopt(int __level, int __optname,
 		}
 	}
 
-	si_tcp_logdbg("going to OS for setsockopt level %d optname %d", __level, __optname);
-	ret = orig_os_api.setsockopt(m_fd, __level, __optname, __optval, __optlen);
-	BULLSEYE_EXCLUDE_BLOCK_START
-	if (ret) {
-		si_tcp_logdbg("setsockopt failed (ret=%d %m)", ret);
+	if (sys_call) {
```

Comment:
The name should be VMA_AVOID_PRIVILEDGED_SYS_CALLS which is better understood than "raw permission". RAW is a different thing than PRIVILEGED